runtime-light/stdlib/diagnostics/exception-types.h

```diff
@@ -89,7 +113,21 @@ inline string exception_trace_as_string(const Throwable& e) noexcept {
 inline void exception_initialize(const Throwable& e, const string& message, int64_t code) noexcept {
   e->$message = message;
   e->$code = code;
-  kphp::log::warning("exception backtrace is not yet supported"); // TODO
+
+  constexpr size_t MAX_BACKTRACE_SIZE = 64;
+  std::array<void*, MAX_BACKTRACE_SIZE> backtrace;
+  const size_t num_frames{kphp::diagnostic::backtrace(backtrace)};
+  std::span<void*> backtrace_view{backtrace.data(), num_frames};
+
+  auto backtrace_symbols{error_handling_impl_::format_backtrace_symbols(backtrace_view)};
+  if (!backtrace_symbols.empty()) {
+    e->trace = std::move(backtrace_symbols);
+  }
+  auto backtrace_code_addresses{error_handling_impl_::format_backtrace_addresses(backtrace_view)};
+  if (!backtrace_code_addresses.empty()) {
+    e->trace = std::move(backtrace_code_addresses);
```

Comment:
Could there be a case where `trace` already has something in it after `format_backtrace_symbols` and we overwrite it here?